script/Arcueid/Day 5/05_00_ARC05_9B.txt

```diff
@@ -372,7 +372,7 @@ The place of death greater than a line.
 [sha:7dcd4a1c96a055fb097c519c83453c55bbace3b5]{
 -- Page 628, Offset 10334.
 -- あの点こそが、線を生み出す死の極点だと。
-That spot must be the locus of death that gives birth to the lines.
+That spot must be the Extremum of death that gives birth to the lines.
```

Comment:
Not sure if I'm using this term the right way

Reply:
We just went over this in the GPR and we've now decided to change this term to "focality" instead of either "locus" or "Extremum"! Sorry about that!

Reply:
I'll also capitalize it based on the spreadsheet